### PR TITLE
Add simple PWA integration

### PR DESCRIPTION
**Pull request by ptesser**
**NO MERGE**

Based on other branch in review. See `feature/front-end/angular-scaffold`.

### CODE REVIEW

File: front-end/src/app/app.module.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { ServiceWorkerModule } from '@angular/service-worker';
+
```
Comment (Vannnu) [inline front-end/src/app/app.module.ts:4]: Line

File: front-end/src/app/app.module.ts [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+import { environment } from '@src/environments/environment';
```
Comment (Vannnu) [inline front-end/src/app/app.module.ts:6]: Line

File: front-end/src/manifest.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      "sizes": "384x384",
+      "type": "image/png"
+    },
+    {
+      "src": "assets/icons/icon-512x512.png",
+      "sizes": "512x512",
+      "type": "image/png"
+    }
+  ]
+}
```
Comment (Vannnu) [inline front-end/src/manifest.json:51]: EOF missing?

**Review by Vannnu — CHANGES_REQUESTED** [review]

File: front-end/src/app/app.module.ts (hunk 2 above)
Comment (ptesser) [reply front-end/src/app/app.module.ts:6]: Because it separates external libs from internals one.

File: front-end/src/app/app.module.ts (hunk 1 above)
Comment (ptesser) [reply front-end/src/app/app.module.ts:4]: Because it separates external libs from internals one.

File: front-end/src/manifest.json (hunk 3 above)
Comment (ptesser) [reply front-end/src/manifest.json:51]: I need to check. I think in a separate JSON file it doesn't matter.

Comment (ptesser) [reply front-end/src/manifest.json:51]: This file is auto created from PWA library.

**Pull request merged**